CHANGES.rst

```diff
@@ -11,6 +11,14 @@
 Changes
 =======
 
+Version 0.2.3 (released 2017-03-09)
+-----------------------------------
+
+Bug fixes
+~~~~~~~~~
+
+- Fixes issue with conditions on extra_require not being taken into account.
```

Comment:
Perhaps:
Fixes **the** issue with ~~conditions~~ markers on extra**S**_require not being taken into account.